packages/react-look/modules/plugins/friendlyClassName.js

```diff
@@ -17,7 +17,13 @@ export default function friendlyClassNames({ StyleContainer, Component, element,
       if (classMapping.has(cls)) {
         className += className + className !== '' ? ' ' : '' + classMapping.get(cls)
       } else {
-        const newClass = classNameTemplate(cls, Component, element)
+        let newClass
+        // checking if config is undefined to avoid breaking existing test
```

Comment:
Actually the interface can be used by people developing plugins (or mixins). The configuration is passed to a wrapper component which passes the config down via context. You could not add a config at all, but the wrapper automatically defaults to {} for those cases. So we're safe to add it :)

If you could do that would be awesome! I will immediately merge afterwards.